docs/source/Regions2D_functionality.ipynb

```diff
@@ -0,0 +1,2342 @@
+{
```

Comment:
<p>can you ad to the first sentence: masks of the sonar data, which indicate which points are within the region.</p><p> </p><p>for the dimension you can use maybe quotes <code>ping_time</code> and <code>depth</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>